backends/avx/ceed-avx-blocked.c

```diff
@@ -1,18 +1,9 @@
-// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
-// Produced at the Lawrence Livermore National Laboratory. LLNL-CODE-734707.
-// All Rights reserved. See files LICENSE and NOTICE for details.
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level COPYRIGHT and NOTICE files for details.
 //
-// This file is part of CEED, a collection of benchmarks, miniapps, software
-// libraries and APIs for efficient high-order finite element and spectral
-// element discretizations for exascale applications. For more information and
-// source code availability see http://github.com/ceed.
+// SPDX-License-Identifier: (BSD-2-Clause)
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-2-Clause
```

My understanding is that parens are only when you have multiple licenses. An official example usage: https://github.com/spdx/spdx-examples/blob/5b8b8b4d533d89959f1a59bae94f2fbc3e0ff14f/example3/content/src/lib.c